assets/js/blocks/cart-checkout/cart/checkout-button/index.js

```diff
@@ -45,6 +52,21 @@ const CheckoutButton = ( { link } ) => {
 			<PaymentMethodIcons
 				icons={ getIconsFromPaymentMethods( paymentMethods ) }
 			/>
+		</>
+	);
+
+	return (
+		<div className="wc-block-cart__submit">
+			{ positionReferenceElement }
+			<div className="wc-block-cart__submit-container">
+				{ submitContainerContents }
+			</div>
+			{ /* If the positionReferenceElement is below the viewport, display the sticky container. */ }
+			{ positionRelativeToViewport === 'below' && (
+				<div className="wc-block-cart__submit-container wc-block-cart__submit-container--sticky">
+					{ submitContainerContents }
+				</div>
+			) }
```

Comment:
That was a bit tricky. If we remove the no-sticky `.wc-block-cart__submit-container` when the sticky version is shown, then `.wc-block-cart__submit` becomes of height=0. In that case, it might become fully visible, so we would remove the sticky container and add again the no sticky one, which might cause `.wc-block-cart__submit` not to be fully visible again. That would cause infinite rerenders.

Hopefully a GIF will explain it better than myself: :slightly_smiling_face: 
![Peek 2020-08-20 10-26](https://user-images.githubusercontent.com/3616980/90743309-f84b1800-e2cf-11ea-824f-d7ab27a94f5f.gif)

To fix that, we need `.wc-block-cart__submit` to always have the same height. I think the cleanest approach to that is to always render the no-sticky container.

I added an explanatory comment in the code in 20b11057e2e9276f25274856df2e223d93760aa1.

Reply:
Works for me 👍 Thanks for the explanation.